### DISCUSSION TITLE
Importing fails, mono is needed.

### DISCUSSION THREAD

**Discussion by mfarhansohail** [Hugovdberg/PIconnect#798] [Q&A]
After installation, the importing PIConnect gives this error

RuntimeError: Failed to create a default .NET runtime, which would
                    have been "mono" on this system. Either install a
                    compatible runtime or configure it explicitly via
                    `set_runtime` or the `PYTHONNET_*` environment variables
                    (see set_runtime_from_env).

I am using databricks to connect

**Comment by Hugovdberg** [comment]
See #619, there is no Linux support for the PI SDK, therefore even if you could have connected to a mono installation you couldn't have run the SDK.
